### PR TITLE
[docker-fpm-frr] limit privileged flag for bgp container

### PR DESCRIPTION
**Pull request by maipbui**
#### Why I did it
HLD implementation: Container Hardening (https://github.com/sonic-net/SONiC/pull/1364)
##### Work item tracking
- Microsoft ADO **(number only)**: 14807420

#### How I did it
Reduce linux capabilities in privileged flag, retain NET_ADMIN and SYS_ADMIN capabilities
#### How to verify it
Install new image to DUT, verify bgp container is up
Run bgp sonic-mgmt kvmtest 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

#### Tested branch (Please provide the tested image version)

- [x] SONiC.frr.0-dirty-20230503.212500
- [ ]

### CODE REVIEW

**Review by qiluo-msft — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

File: rules/docker-fpm-frr.mk [hunk 1]
```diff
@@ -28,7 +28,7 @@ SONIC_DOCKER_IMAGES += $(DOCKER_FPM_FRR)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_FPM_FRR_DBG)
 
 $(DOCKER_FPM_FRR)_CONTAINER_NAME = bgp
-$(DOCKER_FPM_FRR)_RUN_OPT += --privileged -t
+$(DOCKER_FPM_FRR)_RUN_OPT += -t --cap-add=NET_ADMIN --cap-add=SYS_ADMIN
```
Comment (Yarden-Z) [inline rules/docker-fpm-frr.mk:31]: Why is sys_admin required here?

Comment (maipbui) [reply rules/docker-fpm-frr.mk:31]: zebra will unexpectedly exited due to missing the required capabilities.

```
May  2 22:36:14.048947 str-s6000-acs-10 INFO bgp#supervisord: zebra privs_init: initial cap_set_proc failed: Operation not permitted
May  2 22:36:14.049851 str-s6000-acs-10 INFO bgp#supervisord: zebra Wanted caps: cap_net_admin,cap_net_raw,cap_sys_admin=p
May  2 22:36:14.050687 str-s6000-acs-10 INFO bgp#supervisord: zebra Have   caps: cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_net_bind_service,cap_net_admin,cap_net_raw,cap_sys_chroot,cap_mknod,cap_audit_write,cap_setfcap=p
May  2 22:36:14.064956 str-s6000-acs-10 INFO bgp#supervisord 2023-05-02 22:36:14,061 INFO exited: zebra (exit status 1; not expected)
```

**Comment by maipbui** [comment]
@StormLiangMS @Yarden-Z could you help review this PR? Thanks.

**Review by Yarden-Z — APPROVED** [review]
Looks good

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Comment by maipbui** [comment]
@StormLiangMS Could you help Approve cherry pick for 202305 branch? It's for fixing nightly test. MSFT ADO: 24981688

**Pull request merged**